### DISCUSSION TITLE
Support for 1-bit Images

### DISCUSSION THREAD

**Discussion by jgarcia2397** [dlemstra/Magick.NET#976] [Help]
I was reading about the difference between Q8 and Q16 versions in the README. I was wondering if either of these versions support reading and writing of 1-bit images (black and white)? Also, if I have an 8-bit coloured image as an input, can I output a converted 1-bit image?

**Comment by dlemstra** [comment]
With `Q8` the maximum bit depth that can be stored is 8 bit, and it's 16 bit with `Q16`. I used stored instead of read because both of them can read images with a higher bit depth but they have a maximum they can store. And because this is the maximum they can both read 1 bit images. And depending on the output format the image can be changed (bileveled) to 1 bit and then saved as 1 bit. If your use case is 1 bit images only I would advise you to use the `Q8` version to reduce the amount of memory that is being used.
